crates/next-dev-tests/tests/integration/next/font-google/basic/input/pages/index.js

```diff
@@ -18,7 +18,7 @@ export default function Home() {
 function runTests() {
   it("returns structured data about the font styles from the font function", () => {
     expect(interNoArgs).toEqual({
-      className: "className__inter_34ab8b4d__77c0d301",
```

Comment:
Is this indicative of some hash instability?

Reply:
I'm pretty sure this is because right now we hash the filepath instead of the full content, and that changed in this PR.